gradle/libs.versions.toml

```diff
@@ -41,14 +41,15 @@ jackson-datatype-jsr310 = { group = 'com.fasterxml.jackson.datatype', name = 'ja
 
 apache-commons = { group = 'org.apache.commons', name = 'commons-lang3', version.ref = 'apache-commons' }
 javax-ws-rs = { group = 'javax.ws.rs', name = 'javax.ws.rs-api', version.ref = 'javax-ws-rs' }
-sl4j = { group = 'org.slf4j', name = 'slf4j-api', version.ref = 'sl4j' }
 jsoup = { group = 'org.jsoup', name = 'jsoup', version.ref = 'jsoup' }
 aws-lambda-java-core = { group = 'com.amazonaws', name = 'aws-lambda-java-core', version.ref = 'aws-lambda-java-core' }
 google-guava = { group = 'com.google.guava', name = 'guava', version.ref = 'google-guava' }
 
+log4j-api = { group = 'org.apache.logging.log4j', name = 'log4j-api', version.ref = 'log4jApi' }
+slf4j-api = { group = 'org.slf4j', name = 'slf4j-api', version.ref = 'slf4j' }
 
 [bundles]
 nvaCommons = ['nva-commons-core', 'nva-commons-apigateway']
 testing = ['junit-jupiter-api', 'junit-jupiter-params', 'junit-jupiter-engine', 'mockito-core', 'hamcrest', 'nva-testutils', 'nva-logutils', 'zalando']
 jackson = ['jackson-core', 'jackson-databind', 'jackson-annotations', 'jackson-datatype-jdk8', 'jackson-datatype-jsr310']
-others = ['apache-commons', 'javax-ws-rs', 'sl4j', 'jsoup', 'aws-lambda-java-core', 'google-guava']
+others = ['apache-commons', 'javax-ws-rs', 'slf4j-api', 'log4j-api', 'jsoup', 'aws-lambda-java-core', 'google-guava']
```

Comment:
"others" provide little description of what is here and also promotes expanding this bundle with more libraries so that developers skip including explicitly what they want and they just add dependencies without thinking

Reply:
rename "others" or remove it. Maybe in a diffrent PR but "others" is problematic both because of its name and because of its contents. It has for example both apache-commons and guava and they are both big libraries. Do we really need both?

Reply:
Will do in different PR. Then I can try to remove apache-commons or guava as well